resin/base_request.py

```diff
@@ -26,6 +26,10 @@ def __init__(self):
         self.token = Token()
         self.util = Util()
 
+    @property
+    def timeout(self):
+        return int(self.settings.get('timeout'))
```

Comment:
@zvin we need to convert the timeout to float since requests accepts float for timeout.

Reply:
Updated